server.js

```diff
@@ -0,0 +1,240 @@
+const express = require('express');
+const cors = require('cors');
+
+const posts = require('./data/helpers/postDb.js');
+const tags = require('./data/helpers/tagDb.js');
+const users = require('./data/helpers/userDb.js');
+
+const port = 5000;
+const server = express();
+server.use(express.json());
+server.use(cors({ origin: 'http://localhost:3000' }));
+
+
+//---------- Get Methods ----------
```

Comment:
I like this method of organizing your CRUD code -- great for repetition and building muscle memory!
I would urge you to keep in mind that it won't always make sense to lay out a server in this way. Sometimes it may make more sense to organize all the requests by the types of data they access. OR it may make sense to organize them in some other way, based on how you plan to access the endpoints from your front-end.
Just some thoughts -- there's nothing wrong with the way you organized it, but different projects may make more sense organized in a different way.